packages/rs-sdk/src/sdk.rs

```diff
@@ -276,7 +275,11 @@ impl Sdk {
         Ok(())
     }
 
-    // TODO: Changed to public for tests
+    /// Get configured Dash Core network type.
+    pub fn core_network(&self) -> Network {
```

Comment:
network for both platform and core always the same so let's call it just network

Reply:
done